### PR TITLE
Add query integration test

### PR DESCRIPTION
**Pull request by julianrex**
This is part 1 of 3 PRs that port https://github.com/mapbox/mapbox-gl-native/pull/15303 to this repo. (See also #318, #319.)

This PR adds a query test, but primarily adds a regression test for comparing accessibility elements - to track improvements to `accessibilityElementAtIndex:`. See https://github.com/mapbox/mapbox-gl-native-ios/issues/220 for the original issue.

### CODE REVIEW

**Comment by jmkiley** [comment]
Noting that I'm seeing a lot of integration tests fail on `master`. Seems to be unrelated to this PR, though.

File: platform/ios/ios.xcodeproj/xcshareddata/xcschemes/Integration Test Harness.xcscheme [hunk 1]
```diff
@@ -36,6 +36,13 @@
             ReferencedContainer = "container:ios.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "MAPBOX_ACCESS_TOKEN"
+            value = "pk.eyJ1IjoianVsaWFucmV4IiwiYSI6ImNrMGp2N3k3ZzAwbWkzbnMzcG1zZ291cmkifQ.6Fv-dxHmzcqHHm_Tlf3x6A"
```
Comment (jmkiley) [inline platform/ios/ios.xcodeproj/xcshareddata/xcschemes/Integration Test Harness.xcscheme:42]: Do we want to hard code a real access token here? Can we use the same script that iosapp uses to pull in the access token from `~/.mapbox`?

Comment (julianrex) [reply platform/ios/ios.xcodeproj/xcshareddata/xcschemes/Integration Test Harness.xcscheme:42]: Oops! Yeah, that shouldn't be there. Can you remove please?

Comment (julianrex) [reply platform/ios/ios.xcodeproj/xcshareddata/xcschemes/Integration Test Harness.xcscheme]: For future reference - this access token has been revoked.

**Pull request merged**